examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -308,13 +311,15 @@ sparkz {
         scId: "1f3b001136ca1afb15e13ac325170929feca6282bdfd89365684bf01f7c6ba14"
         powData: "1dd1b161f70e0f201dd1b161f70e0f201dd1b161f70e0f201dd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201cd1b161f70e0f201bd1b161f70e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201bd1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f201ad1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f2019d1b161f80e0f20"
-        mcBlockHeight: 450
+        mcBlockHeight: 480
         mcNetwork: regtest
+        isNonCeasing: false
         withdrawalEpochLength: 900
         initialCumulativeCommTreeHash: "68ed60c2e761bca39e2acb0b97bc1d6cab3c4e34e9418e315a0a152c092da60e"
     }
 }
 ```
+_For non ceasing sidechains withdrawalEpochLength parameter has value of virtualEpochLength._
```

Comment:
I think we miss the change about the `virtualEpochLength` usage in one of the BootTool commands for non-ceasing sidechains

Reply:
Done. Also duplicated to circuitWithKeyRotation branch.